controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -465,10 +465,10 @@ void cli_bml::setFunctionsMapAndArray()
         "bml_restricted_channels_get_slave", "<mac>", "Get restricted channels from slave 'mac'",
         static_cast<pFunction>(&cli_bml::get_slave_restricted_channels_caller), 1, 1, STRING_ARG);
     insertCommandToMap(
-        "bml_wfca_controller", "<string_command>", "send wfca controller<string_command>",
-        static_cast<pFunction>(&cli_bml::bml_wfca_controller_caller), 1, 1, STRING_ARG);
```

Comment:
This function was already renamed in the previous commit, so the previous commit is broken.

That's why I said that you should first do renames and only afterwards add new code :-).

@tomereli is it necessary to fix this?

Reply:
we better fix it, yes. I'll help Moran.